python/pyenv/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    pyenv
+version                 2.3.2
+categories-append       python devel
+license                 MIT
+maintainers             nomaintainer
+supported_archs         noarch
+description             Simple Python Version Management
+long_description        pyenv lets you easily switch between multiple versions of Python. It's simple, unobtrusive, and follows the UNIX tradition of single-purpose tools that do one thing well.
+homepage                https://github.com/pyenv/pyenv
+master_sites            https://github.com/pyenv/pyenv/archive/refs/tags/v${version}.tar.gz
+checksums               rmd160 28cb721fcd3d3fbdb95d44708e5d773e7bacab01 \
+                        sha256 150ac8f7161c00e8e613bf5e273306f674b60037e3dace9c6bb7611dceb17144 \
+                        size   1400
+depends_lib             port:autoconf \
```

Comment:
where do these dependencies come into play? For just being a "shim" for different Python versions this list seemsa little weird to me...

Reply:
They're based around providing dependencies for subsequent installs of Python - not essential but they do mean you avoid potential "weird" issues when pyenv compiles the Python version you're bringing in https://github.com/pyenv/pyenv/wiki#suggested-build-environment

Reply:
ah, now I remember.... that's the part I personally don't like about this package.... We should use MacPorts to install Python versions not some third-party tool ;)

Reply:
Sadly people on dev teams insist on using different OS's and package managers, they have not seen the light. 😭 😂

Reply:
That said, I am running a steady campaign to get my current team to all move to MacPorts…